### ISSUE TITLE
Reg\Events - MVP: External User has an Instruction Page for reporting closures/transfers

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#2374]
#### Dependencies

- Blocked by
- Blocking

#### Description:
For the MVP, as an external user, I have a Report a Closure/Transfer tile in my Registration dashboard and when I click on it, it takes me to an instruction page where I can see how to report  

#### Acceptance Criteria:

Given I am on the [Administration dashboard](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=22844-976&t=ccI6nsQrfaNlSsbJ-4) OR I am on the [Main BCIERS dashboard](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=21269-5893&t=ccI6nsQrfaNlSsbJ-4) 
When I click on Report a Closure/Transfer
Then it takes me to this Instruction Page: [Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=23217-5751&t=CcU7KP90YNmWgojg-4)

The instruction page will link to the following CHEFS form for the user to fill out and report an event. https://submit.digital.gov.bc.ca/app/form/submit?f=d26fb011-2846-44ed-9f5c-26e2756a758f

#### Development Checklist:

- [x] add the new tile to the registration dashboard.json for external users, both industry_user and industry_user_admin
- [x] add a link to the BCIERS dashboard
- [x] create a new page.tsx to display the instructions
- [x] link the tiles to the instructions
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by zoeyli-46** [comment]
@patriciarussellCAS @andrea-williams @shon-button I made this ticket for the instruction page that users get when they click on the report closure/transfer tile

**Comment by patrickisaac** [comment]
Refinement Notes:
- @BCerki will provide Dev checklist

**Comment by BCerki** [comment]
@zoeyli-46 , the wireframes etc. are for the registration-specific dashboard. Are we showing anything about closures/transfers on the main BCIERS dashboard? (this one: ![Image](https://github.com/user-attachments/assets/fd9ec341-5259-4432-bae8-a6ffca22d150)
)

**Comment by zoeyli-46** [comment]
Hey @BCerki 
1. Yes, both admin and reporter will be seeing the ability to report a closure/transfer. 
2. https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=21269-5893&t=ccI6nsQrfaNlSsbJ-4 It will be a link to show up on the Administration tile under Operations

**Comment by andrea-williams** [comment]
do we have a link to the form (mentioned in the wireframe) yet, or should we just be inserting a placeholder for now?

**Comment by zoeyli-46** [comment]
They have not created the CHEFS form yet, we can use a placeholder for now.

**Comment by BCerki** [comment]
@zoeyli-46 , I think the figma link to the main dashboard might be outdated. I don't see a transfer tile on it: 

![Image](https://github.com/user-attachments/assets/71d7417c-c64c-436a-8787-58d89cd4ed23)

**Comment by patriciarussellCAS** [comment]
It looks like on the first wireframe, Transfers are now under Admin. I think that makes sense, rather than Registration. I"ll let zoey confirm but I assume it should live under admin.

**Comment by BCerki** [comment]
OK, gotcha! There's not a separate tile on the main dashboard, just the link under admin. Thanks!

**Comment by zoeyli-46** [comment]
@BCerki Thats correct!

**Comment by fviduya** [comment]
I have tested this in the DEV environment. [Here is the test document for reference](https://bcgov-my.sharepoint.com/:w:/g/personal/francesca_viduya_gov_bc_ca/ERyHg8KI12BHqm0OpsZwQlcBc-6rj0fsfFJPow9aN9LG3g?e=6kvnIX).

I can confirm that 
- [x] ```Report transfer of operation or facility``` is visible in the Main BCIERS dashboard/Administration dashboard
- [x] Clicking ```Report transfer of operation or facility``` redirects to the instruction page as specified in the wireframe
- [x] Clicking ```Report an Event``` redirects to the [CHEFS form](https://submit.digital.gov.bc.ca/app/form/submit?f=d26fb011-2846-44ed-9f5c-26e2756a758f) which allows user to fill out a form and report an event.

**Comment by patriciarussellCAS** [comment]
Reviewed QA test document and will approve this ticket. I'm ok with the breadcrumbs only showing "transfers" for now, but it would be nice to update this later if/when there is time.
